src/screens/MemberDetail/MemberDetail.tsx

```
@@ -271,224 +274,476 @@
+                          />
                         )}
-                        alt="User"
-                        data-testid="profile-picture"
-                        crossOrigin="anonymous" // to avoid Cors
+                        <i
+                          className="fas fa-edit position-absolute bottom-0 right-0 p-2 bg-white rounded-circle"
+                          onClick={() => fileInputRef.current?.click()}
+                          data-testid="uploadImageBtn"
+                          style={{ cursor: 'pointer', fontSize: '1.2rem' }}
+                          title="Edit profile picture"
+                          role="button"
```

Biome (2.1.2)

[error] The HTML element i is non-interactive and should not have an interactive role.

Replace i with a div or a span.
Unsafe fix: Remove the role attribute.


(lint/a11y/noNoninteractiveElementToInteractiveRole)